samples/dashboards/demo/datagrid-mathmodifier/demo.html

```diff
@@ -7,12 +7,12 @@
 
 <div id="container">
 
-<pre id="csv">Day, EUR, Rate
-15, 11, 1.0876
-16, 23, 1.0881
-17, 15, 1.0829
-18, 27, 1.0813
-19, 13, 1.0808</pre>
+<pre id="csv">Day,EUR,Rate
+15,11,1.0876
+16,23,1.0881
+17,15,1.0829
+18,27,1.0813
+19,13,1.0808</pre>
```

Comment:
Why changing this? 🤔

Reply:
with spaces, the name of the series will be ` EUR` with the extra space in the beginning, which may cause some bugs, when we will try to test something in the future. I thought that a good idea would be to fix it here by the way.